ui/__tests__/index.test.tsx

```diff
@@ -1,13 +1,34 @@
 import TodosPage from 'pages/index'
 import { render, screen } from '@testing-library/react'
-import { Story } from '@prisma/client'
+import { Story } from 'domain-logic/stories'
+
+jest.mock('swr', () => {
+  return () => {
+    const pendingStory: Story = {
+      id: 'some id',
+      asA: 'developer',
+      iWant: 'write test files',
+      soThat: 'I have confidence',
+      state: 'pending',
+      createdAt: new Date(),
+    }
+
+    return { data: [pendingStory] }
+  }
+})
 
 describe('TodosPage', () => {
   it('renders', () => {
-    render(
-      <TodosPage/>
-    )
+    render(<TodosPage />)
 
     expect(screen.getByText(/Stories/)).toBeInTheDocument()
   })
+
+  it('renders story titles', () => {
+    render(<TodosPage />)
+
+    expect(
+      screen.getAllByText(/write test files/).length
+    ).toBeGreaterThanOrEqual(1)
```

Comment:
@diogob , any special reason for asserting with such imprecise method?

Reply:
this text shows more than once, once in the story title and once in the description.
I'll use a matcher to get only the title instance

Reply:
Maybe we could change the title of the stories from `span` to some heading like `h4` for instance. Then you'll be able to select it with `screen.getByRole('heading', { name: /write test files/i })`

Reply:
I have tightened the expectation by requiring 2 ocurrences. Later we can improve on the matcher if we need something more precise